### ISSUE TITLE
Show seconds along with the minutes in schedule focus

### ISSUE THREAD

**Issue by mehedihasan2810** [tijlleenders/ZinZen#1834]
**Is your feature request related to a problem? Please describe.**
It's very boring when we start the focus timer and see no changes before the next min

**Describe the solution you'd like**
Show seconds along with the minutes. 

The quick changes in the UI will help us stay focus not bored

**Describe alternatives you've considered**
none

**Additional context**

https://github.com/tijlleenders/ZinZen/assets/117534561/1157c086-c52e-4697-ada3-1b37d00ca530

**Comment by tijlleenders** [comment]
@mehedihasan2810 I do think you're on to something - but I don't think showing seconds has the right effect. It can also be a bit like the bomb-timer in the movies...  

Maybe a slowly changing color blur or slow moving circle?

**Comment by mehedihasan2810** [comment]
Yep you are right.

I think slowly moving the circle with a little bit of animation/transition would be great.

if we are agree with this one then I can take a look. Let ne know

@tijlleenders

**Comment by mehedihasan2810** [comment]
Just checked the slow moving circle one is the current behavior

**Comment by tijlleenders** [comment]
> Just checked the slow moving circle one is the current behavior.

Yeah - but the movement is not really noticeable. It could have like a glow going through it slowly or something. Nothing too busy - but something to indicate the timer is active.


The done at the end is also nice.

Sure - if you want to try this, please do!

**Comment by tijlleenders** [comment]
Closing this due to inactivity @mehedihasan2810 

Thanks for trying - and feel free to re-open if you have time.
